tests/acceptance/features/cliDataExporter/import.feature

```diff
@@ -21,3 +21,56 @@ Feature: An administrator wants to import a user using the commandline
     And as "usertrash" file "AFolder/DeletedFile.txt" should exist
     And the content of file "AFolder/DeletedFolder/fileinfolder.txt" for user "usertrash" should be "text in file in deleted folder"
     And the content of file "AFolder/DeletedFile.txt" for user "usertrash" should be "text in deleted file"
+
+  Scenario: Restore and rename files in trashbin after import
+    Given a user has been imported from path "trashbinExport/usertrash" using the occ command
+    And the administrator has changed the password of user "usertrash" to "123456"
+    When user "usertrash" restores the folder with original path "AFolder/DeletedFolder" using the trashbin API
+    And user "usertrash" restores the file with original path "AFolder/DeletedFile.txt" using the trashbin API
+    And as "usertrash" file "AFolder/DeletedFolder/fileinfolder.txt" should exist
+    And as "usertrash" file "AFolder/DeletedFile.txt" should exist
```

Comment:
```suggestion
    Then as "usertrash" file "AFolder/DeletedFolder/fileinfolder.txt" should exist
    And as "usertrash" file "AFolder/DeletedFile.txt" should exist
```

Reply:
@swoichha are you going to make this change?
The rest of it looks good.

Reply:
I just made the change.